MonsterWaves/Screen/SplashScreen.cpp

```diff
@@ -0,0 +1,55 @@
+#include "SplashScreen.hpp"
+
+SplashScreen::SplashScreen(float duration)
```

Comment:
@JakubMastalski to nie zadziała. Każdy ekran musi mieć dostęp do swojego ekranu.